channeldb/migration_01_to_11/db.go

```diff
@@ -318,7 +227,7 @@ func createChannelDB(dbPath string) error {
 		}
 
 		meta := &Meta{
-			DbVersionNumber: getLatestDBVersion(dbVersions),
```

Comment:
hm, can't this code be removed?

Reply:
Always set to 0 now. It is just for migration testing